### DISCUSSION TITLE
Set themse based on Language/FileType

### DISCUSSION THREAD

**Discussion by dream-dasher** [zed-industries/zed#21419] [Feature Requests]
It's very nice to have a toml file and a rust file and a .env file and a python file all use different themes.

I had a fack in neovim where I could change theme whenever an LSP was activated (not perfect, but at least differentiated work being done in python vs lua vs rust, for example).

Is there anything similar here?
It's a very nice feature to have.
I didn't see anything in settings that connected LSPs or files to themes, nor anything in experimental.theme_override that accounted for LSP/Language/FileType

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
